runway/module/cloudformation.py

```diff
@@ -14,42 +13,6 @@
 LOGGER = logging.getLogger('runway')
 
 
-def ensure_stacker_compat_config(config_filename):
```

Comment:
Moved these inside the class to take advantage of `folder` and to remove some params.